salt/cloud/clouds/gce.py

```diff
@@ -369,9 +369,12 @@ def __get_image(conn, vm_):
     The get_image for GCE allows partial name matching and returns a
     libcloud object.
     """
+    default_img = "debian-7"
     img = config.get_cloud_config_value(
-        "image", vm_, __opts__, default="debian-7", search_global=False
+        "image", vm_, __opts__, default=default_img, search_global=False
     )
+    if not img or not isinstance(img, str):
```

Comment:
The `get_cloud_config_value` should return `debian-7` if it can't find image set anywhere. Can you elaborate why you are adding this if statement here?

Reply:
Added purely to get [this test case](https://github.com/saltstack/salt/pull/62046/files/5cd5fd75bb7eb7a60e6d73d4efc7611e482ebff9#diff-011c0910037d4cbee4d337f9311903470b81753065ce3586eff691d536f5a465R55) to pass. Could remove it and the case

Reply:
I would prefer to remove the if statement and fix the test. We should be able to mock what is returned by `config.get_cloud_config_value`